SharedTests/UtilsTests.swift

```diff
@@ -46,18 +46,6 @@ class UtilsTests: XCTestCase {
         XCTAssertFalse(optArrayEqual(z, rhs: y))
     }
 
-    func testWithQueryParam() {
-        let urlA = NSURL(string: "http://foo.com/bar/")!
-        let urlB = NSURL(string: "http://bar.com/noo")!
-        let urlC = urlA.withQueryParam("ppp", value: "123")
-        let urlD = urlB.withQueryParam("qqq", value: "123")
-        let urlE = urlC.withQueryParam("rrr", value: "aaa")
-
-        XCTAssertEqual("http://foo.com/bar/?ppp=123", urlC.absoluteString)
-        XCTAssertEqual("http://bar.com/noo?qqq=123", urlD.absoluteString)
-        XCTAssertEqual("http://foo.com/bar/?ppp=123&rrr=aaa", urlE.absoluteString)
-    }
-
```

Comment:
Where? I see tests removed here and in `AboutUtilsTests.swift`, but I don't see tests added anywhere in this PR.